gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergDatasetFinder.java

```diff
@@ -125,14 +126,14 @@ protected IcebergCatalog createIcebergCatalog(Properties properties, CatalogLoca
     switch (location) {
       case SOURCE:
         catalogUri = properties.getProperty(ICEBERG_SRC_CATALOG_URI_KEY);
-        Preconditions.checkNotNull(catalogUri, "Source Catalog Table Service URI is required");
+        Preconditions.checkNotNull(catalogUri, "Provided: {%s} Source Catalog Table Service URI is required", catalogUri);
```

Comment:
Correct me if I'm wrong here, but wouldn't this message always equate to:
```
Provided: null Source Catalog Table Service URI is required
```

I think I just wanted you to provide the ICEBERG_SRC_CATALOG_URI_KEY

Reply:
gotcha... updated the error message in my latest commit!